packages/components/src/organisms/AddReviewModal/AddReviewModal.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import { Modal, ModalHeader, useUI } from '../..'
+import type { ModalProps, OverlayProps } from '../..'
+
+export interface AddReviewModalProps extends ModalProps {
```

Comment:
Can't we call this component just `ReviewModal`? 🤔

Reply:
I think we can just use ReviewModal and also for the other variables. 
@renatamottam WDYT?